.drone.yml

```diff
@@ -1,78 +1,469 @@
+    key:
+      from_secret: gpgsign_key
+    passphrase:
+      from_secret: gpgkey_passphrase
+  when:
+    event:
+    - tag
+
+- name: github
+  pull: always
+  image: plugins/github-release
+  settings:
+    files:
+    - "release/*.tar.gz"
+    - "release/*.tar.gz.sha256"
+    - "release/*.tar.gz.asc"
+    token:
+      from_secret: github_token
+  when:
+    event:
+    - tag
+
+trigger:
+  ref:
+  - refs/heads/master
+  - "refs/tags/**"
+  - "refs/pull/**"
+
+depends_on:
+- testing
+
+---
+kind: pipeline
+name: linux-arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+- name: build-push
+  pull: always
+  image: golang:1.11
+  commands:
+  - "go build -v -ldflags \"-X main.version=${DRONE_COMMIT_SHA:0:8}\" -a -tags netgo -o release/linux/arm64/drone-autoscaler ./cmd/drone-autoscaler"
+  environment:
+    CGO_ENABLED: 0
+    GO111MODULE: on
+  when:
+    event:
+      exclude:
+      - tag
+
+- name: build-tag
+  pull: always
+  image: golang:1.11
+  commands:
+  - "go build -v -ldflags \"-X main.version=${DRONE_TAG##v}\" -a -tags netgo -o release/linux/arm64/drone-autoscaler ./cmd/drone-autoscaler"
+  environment:
+    CGO_ENABLED: 0
+    GO111MODULE: on
+  when:
+    event:
+    - tag
+
+- name: dryrun
+  pull: always
+  image: plugins/docker:linux-arm64
+  settings:
+    daemon_off: false
+    dockerfile: docker/Dockerfile.linux.arm64
+    dry_run: true
     password:
       from_secret: docker_password
+    repo: drone/autoscaler
+    tags: linux-arm64
     username:
       from_secret: docker_username
   when:
     event:
-    - push
+    - pull_request
+
+- name: publish
+  pull: always
+  image: plugins/docker:linux-arm64
+  settings:
+    auto_tag: true
+    auto_tag_suffix: linux-arm64
+    daemon_off: false
+    dockerfile: docker/Dockerfile.linux.arm64
+    password:
+      from_secret: docker_password
+    repo: drone/autoscaler
+    username:
+      from_secret: docker_username
+  when:
+    event:
+      exclude:
+      - pull_request
+
+- name: tarball
+  pull: always
+  image: golang:1.11
+  commands:
+  - tar -cvzf release/autoscaler_linux_arm64.tar.gz -C release/linux/arm64 autoscaler
+  - "sha256sum release/autoscaler_linux_arm64.tar.gz > release/autoscaler_linux_arm64.tar.gz.sha256"
+  when:
+    event:
     - tag
 
-services:
-- name: postgres
-  pull: default
-  image: postgres:9
+- name: gpgsign
+  pull: always
+  image: plugins/gpgsign
+  settings:
+    files:
+    - "release/*.tar.gz"
+    - "release/*.tar.gz.sha256"
+    key:
+      from_secret: gpgsign_key
+    passphrase:
+      from_secret: gpgkey_passphrase
+  when:
+    event:
+    - tag
+
+- name: github
+  pull: always
+  image: plugins/github-release
+  settings:
+    files:
+    - "release/*.tar.gz"
+    - "release/*.tar.gz.sha256"
+    - "release/*.tar.gz.asc"
+    token:
+      from_secret: github_token
+  when:
+    event:
+    - tag
+
+trigger:
+  ref:
+  - refs/heads/master
+  - "refs/tags/**"
+  - "refs/pull/**"
```

Comment:
these should be removed because it prevents me from testing feature branches.  I understand this is a standard for drone-plugins, but is not a standard I want to impose in `drone/` org.

Reply:
I will drop that expect for the release pipeline steps